### PR TITLE
Add news section and initial post

### PR DESCRIPTION
**Pull request by pvanderknyff**
I need to beef up the post a bit, but this adds a news section that we can post to. I've tested locally, will try and get it to be visible to other computers as well. You can get a good idea of what it looks like from here (since I borrowed a lot of code from it): https://tableau.github.io/webdataconnector/news/

### CODE REVIEW

File: _layouts/news.html [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+<body>
+    <div class="container">
+        {% include header.html %}
+        <div>
+		<h3>Connector SDK News</h3>
+		    <ul class=posts >
+			  {% for post in site.posts %}
+	          <li><a href="#{{ post.id }}" >{{ post.title }}</a> &nbsp;  <span style="font-size:80%;">({{ post.date | date_to_string }})</span></li>
+	         {% endfor %}
```
Comment (joe-langford) [inline _layouts/news.html:19]: nit: endfor vertical align

File: _layouts/news.html [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+            <div>
+                <h1 id="{{post.id}}">{{ post.title }} </a></h1>
+                <hr>
+                <p><span class="glyphicon glyphicon-time"></span> Posted on {{ post.date | date: "%-d %B %Y" }}</p>
+                <hr>
+                <p class="lead">
+                    {{ post.abstract }}
+                </p>
+                {% if post.photoname %}
+                <img class="img-responsive" src="{{ site.baseurl }}/assets/{{ post.photoname}}" alt=""> {% endif %}
```
Comment (joe-langford) [inline _layouts/news.html:34]: nit: endif on next line

File: _posts/2020-03-31-new-release.md [hunk 3]
```diff
@@ -0,0 +1,6 @@
+---
+title:  "Tableau Connector SDK for 2019.4 Released, latest version of SDK now targets 2020.1"
+abstract: "The Connector SDK for 2019.4 has released for those wishing to target that version. The main SDK branch now targets Tableau 2020.1"
+---
+
+On March 11th, with the full release of Tableau 2020.1 we moved the latest version of the master branch to target Tableau 2020.1. This means that the samples and connectors made with this version of the SDK may not work with older versions of Tableau. To develop against Tableau 2019.4, please download the "Tableau Connector SDK for Tableau 2019.4" release.
```
Comment (rosswbrown) [inline _posts/2020-03-31-new-release.md:6]: If possible can you do a markdown link for "Tableau Connector SDK for Tableau 2019.4" and remove quotes.

Comment (lukewrites) [reply _posts/2020-03-31-new-release.md:6]: can also make that a link to the release maybe?

File: _layouts/news.html (hunk 2 above)
Comment (pvanderknyff) [reply _layouts/news.html]: Done

File: _layouts/news.html (hunk 1 above)
Comment (pvanderknyff) [reply _layouts/news.html:19]: It should be aligned now

File: _posts/2020-03-31-new-release.md (hunk 3 above)
Comment (pvanderknyff) [reply _posts/2020-03-31-new-release.md]: Added link, and a bit on vendor attributes as well

**Pull request merged**